tests/DevBetterWeb.Tests/Core/Entities/MemberTests/MemberUpdateAboutInfo.cs

```diff
@@ -68,5 +69,37 @@ public void RecordsEventWithAppendedDetailsIfOtherThingsChanged()
             Assert.Equal("Name,AboutInfo", eventCreated.UpdateDetails);
         }
 
+        [Fact]
+        public void ReturnsInputYouTubeUrlIfContainsQuestionMark()
+        {
+            var member = MemberHelpers.CreateWithDefaultConstructor();
+
+            member.UpdateLinks(null, null, null, null, null, "https://www.youtube.com/ardalis?", null);
+
+            var eventCreated = (MemberUpdatedEvent)member.Events.First();
+
+            MemberLinksDTO dto = MemberLinksDTO.FromMemberEntity(member);
+
+            var result = dto.YouTubeUrl;
+
+            Assert.Equal("https://www.youtube.com/ardalis?", result);
```

Comment:
Don't repeat magic string. Use a variable for this value both here and on line 77.